index.html

```diff
@@ -69,5 +70,185 @@ <h2 class="text-2">
+               <li class="button3">html</li>
+               <li class="button3">bootstrap</li>
+               <li class="button3">Ruby</li>                                                    
+            </ul>            
+           <div class="orange-button">
+               See Project
+           </div>                                
+        </div>
+
+        <div class="card card-5">
+            <div class="title-post"> Profesional Art Printing Data </div>
+            <div class="supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+
+            <ul class="button-container-float">
+               <li class="button3">html</li>
+               <li class="button3">bootstrap</li>
+               <li class="button3">Ruby</li>                                                    
+            </ul>            
+           <div class="orange-button">
+               See Project
+           </div>                                
+        </div>
+
+        <div class="card card-6">
+            <div class="title-post"> Profesional Art Printing Data </div>
+            <div class="supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+
+            <ul class="button-container-float">
+               <li class="button3">html</li>
+               <li class="button3">bootstrap</li>
+               <li class="button3">Ruby</li>                                                    
+            </ul>            
+           <div class="orange-button">
+               See Project
+           </div>                                
+        </div>
+
+
+    </div>    
+
+    <div class="wrapper-space-white"></div>
+
+    <section class="finale">
+        
+        <h2 class="about-me">
+            About me
+        </h2>
+        <p class="finale-description">
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="get-my-resume">
+            Get My Resume
+        </button>                
+    </section>
+    
+    <div class="wrapper-end">
+        <div class="line-separator">ss</div>
+
+        <div class="container-duo">
+            <div class="item-rotated-square">  </div>
+            <div class="item-2"> Language </div>
+        </div>
+
+        <ul class="list-arrange">
+            <li class="language-item">Java Script</li>
+            <li class="language-item">Ruby</li>
+            <li class="language-item">Html</li>
+            <li class="language-item">Css</li>
+        </ul>        
+
+        <div class="container-duo">
+            <div class="item-square">  </div>
+            <div class="item-2"> Frameworks </div>
+        </div>
```

Comment:
- [ ] The same for the Frameworks, it will be better to replace the `div` by an `h2`